NOTICE

```diff
@@ -33,3 +33,6 @@ the following copyright notice:
 
 Apache Spark
 Copyright 2014 and onwards The Apache Software Foundation.
+
+PyHive
+This project includes code from PyHive originally developed by Dropbox and was donated to Apache/Kyuubi in 2024
```

Comment:
could you please eliminate "and was donated to Apache/Kyuubi in 2024", it's a little bit redundant and I haven't seen such words in other project NOTICE.

Reply:
@pan3793 that was the ask from our legal department. Is it ok to keep this phrasing ?

Reply:
Okay